### PR TITLE
Remove non-working connectors #-E

### LINKED ISSUE #2321: Connector checkup #-E
Connector | URL | Automatic Test | Europe | Asia | US
----- | ----- | ----- | ----- | ----- | ----- 
.MangaRock.mjs|https://mangarock.com|✔️|❌|❌|❌
.RawQQ.mjs|https://rawqq.com|❌|❌|❌|❌
ActionManga.mjs|https://www.action-manga.com|❌|❌|❌|❌
AdonisFansub.mjs|https://manga.adonisfansub.com|403|✔️|(✔️)|✔️
AHStudios.mjs|https://ahstudios.net|✔️|✔️|(✔️)|✔️
AjiaNoScantrad.mjs|https://ajianoscantrad.fr|❌|❌|❌|❌
AkaiYuhiMun.mjs|https://akaiyuhimun.ru|403|1020|1020|1020
AlianzaMarcial.mjs|https://www.alianzamarcial.xyz|❌|❌|❌|❌
AllHentai.mjs|http://allhentai.ru|✔️|✔️|(✔️)|✔️
AllManga.mjs|https://mangaarab.com|❌|❌|❌|❌
AllPornComic.mjs|https://allporncomic.com|✔️|✔️|(✔️)|✔️
AnataNoMotokare.mjs|http://motokare.xyz|403|✔️|(✔️)|✔️
AngelOfResistance.mjs|https://angelofresistance-tr.com|✔️|Susp.|Susp.|Susp.
AnimePahe.mjs|https://animepahe.com|✔️|✔️|(✔️)|✔️
AnimExtremist.mjs|http://www.animextremist.com|✔️|✔️|(✔️)|✔️
AniRune.mjs|https://anirune.com|✔️|✔️|(✔️)|✔️
Anivizyon.mjs|https://www.anivizyon.com|✔️|✔️|(✔️)|✔️
AntisenseScans.mjs|http://antisensescans.com|406|✔️|(✔️)|✔️
AOC.mjs|https://aoc.moe|❌|❌|❌|❌
ApollComics.mjs|https://apollcomics.xyz|✔️|✔️|(✔️)|✔️
ApollToons.mjs|https://apolltoons.xyz|✔️|✔️|(✔️)|✔️
ArangScans.mjs|https://arangscans.com|✔️|✔️|(✔️)|✔️
ArazNovel.mjs|https://www.araznovel.com|✔️|✔️|(✔️)|✔️
ArgosScan.mjs|https://argosscan.com|✔️|✔️|(✔️)|✔️
ArtemisNoFansub.mjs|https://artemisnf.com|522|522|522|522
AsmHentai.mjs|https://asmhentai.com|✔️|✔️|(✔️)|✔️
AstralLibrary.mjs|https://astrallibrary.net|✔️|✔️|(✔️)|✔️
AsuraScans.mjs|https://asurascans.com|✔️|✔️|(✔️)|✔️
AtelierDuNoir.mjs|http://atelierdunoir.org|404|❌|❌|❌
Atikrost.mjs|https://atikrost.com|✔️|✔️|(✔️)|✔️
AtmSubs.mjs|https://atm-subs.fr|✔️|❌|(✔️)|✔️
Authrone.mjs|http://www.authrone.com|✔️|❌|❌|❌
AzoraManga.mjs|https://www.azoramanga.com|✔️|✔️|(✔️)|✔️
BacaKomik.mjs|https://bacakomik.co|✔️|✔️|(✔️)|1020
BacaManga.mjs|https://bacamanga.co|503|✔️|(✔️)|✔️
BakaguyaScan.mjs|https://bakaguya-scan.tk|❌|❌|❌|❌
Batoto.mjs|https://bato.to|✔️|✔️|(✔️)|✔️
BestManga.mjs|https://bestmanga.club|✔️|✔️|(✔️)|✔️
Biamam.mjs|http://biamam.com|✔️|✔️|(✔️)|✔️
BlackDragonsFansub.mjs|https://bdsfansub.com|✔️|✔️|(✔️)|✔️
BlogTruyen.mjs|https://blogtruyen.vn|❌|✔️|(✔️)|❌
BookLive.mjs|https://booklive.jp|✔️|✔️|(✔️)|✔️
BoredomSociety.mjs|https://www.boredomsociety.xyz|✔️|httpd2|httpd2|httpd2
BunnysScans.mjs|http://bns.shounen-ai.net/read/|❌|❌|❌|❌
CanisMajorScans.mjs|http://cm-scans.shounen-ai.net|❌|❌|❌|❌
CentralDeMangas.mjs|http://cdmnet.com.br|✔️|✔️|❌|✔️
ChampionScans.mjs|https://read.ptscans.com|✔️|✔️|(✔️)|✔️
ChibiManga.mjs|http://www.cmreader.info|✔️|✔️|(✔️)|✔️
CiayoEN.mjs|https://www.ciayo.com/en|❌|❌|❌|❌
CiayoID.mjs|https://www.ciayo.com/id|❌|❌|❌|❌
CloverManga.mjs|https://clover-manga.com|✔️|✔️|(✔️)|✔️
ComicAction.mjs|https://comic-action.com|✔️|✔️|(✔️)|✔️
ComicBoost.mjs|https://comic-boost.com|✔️|✔️|(✔️)|✔️
ComicBorder.mjs|https://comicborder.com|✔️|✔️|(✔️)|✔️
ComicDays.mjs|https://comic-days.com|✔️|✔️|(✔️)|✔️
ComicEarthStar.mjs|https://www.comic-earthstar.jp|✔️|✔️|(✔️)|✔️
ComicExtra.mjs|http://www.comicextra.com|✔️|✔️|(✔️)|✔️
ComicFire.mjs|https://www.hobbyjapan.co.jp/comic|❌|❌|(✔️)|❌
ComicGardo.mjs|https://comic-gardo.com|✔️|✔️|(✔️)|✔️
ComicMeteor.mjs|https://comic-meteor.jp|✔️|✔️|(✔️)|✔️
Comico.mjs|https://www.comico.jp|✔️|✔️|(✔️)|✔️
ComicoNovel.mjs|https://novel.comico.jp|✔️|✔️|(✔️)|✔️
ComicRyu.mjs|https://www.comic-ryu.jp|✔️|✔️|(✔️)|✔️
ComicVN.mjs|https://beeng.net|✔️|✔️|(✔️)|✔️
ComicWalker.mjs|https://comic-walker.com|✔️|✔️|(✔️)|✔️
CopyPasteScan.mjs|https://copypastescan.xyz|✔️|✔️|(✔️)|✔️
DangolineNoFansub.mjs|http://lector.dangolinenofansub.com|✔️|_empty_|_empty_|_empty_
DarkSkyProjects.mjs|https://darkskyprojects.org|503|✔️|(✔️)|✔️
DarkYueRealm.mjs|https://darkyuerealm.site|406|✔️|(✔️)|✔️
DaumWebtoon.mjs|http://webtoon.daum.net|✔️|✔️|(✔️)|✔️
DeathTollScans.mjs|https://reader.deathtollscans.net|✔️|✔️|(✔️)|✔️
DemoneCeleste.mjs|https://www.demoneceleste.it|❌|✔️|(✔️)|✔️
Desu.mjs|https://desu.me|✔️|✔️|(✔️)|✔️
DigitalTeam.mjs|https://dgtread.com|✔️|✔️|(✔️)|✔️
DisasterScans.mjs|https://disasterscans.com|✔️|✔️|(✔️)|✔️
DokiFansubs.mjs|https://kobato.hologfx.com|✔️|✔️|(✔️)|✔️
DongManManhua.mjs|https://www.dongmanmanhua.cn|✔️|✔️|(✔️)|❌
DoujinDesu.mjs|https://doujindesu.info|✔️|✔️|(✔️)|✔️
DoujinHentai.mjs|https://doujinhentai.net|✔️|✔️|(✔️)|✔️
Doujins.mjs|https://doujins.com|✔️|✔️|(✔️)|✔️
DoujinYosh.mjs|https://doujinyosh.work|✔️|❌|❌|❌
DropeScan.mjs|https://dropescan.com|✔️|✔️|(✔️)|✔️
DynastyScans.mjs|https://dynasty-scans.com|✔️|✔️|(✔️)|✔️
EarlyManga.mjs|https://earlymanga.website|✔️|✔️|(✔️)|✔️
EcchiScan.mjs|https://ecchiscan.com|522|522|522|522
EdelgardeScans.mjs|https://edelgardescans.com|✔️|✔️|(✔️)|✔️
EGScans.mjs|https://read.egscans.com|❌|❌|❌|❌
EHentai.mjs|https://e-hentai.org|✔️|✔️|(✔️)|✔️
EightMuses.mjs|https://comics.8muses.com|✔️|✔️|(✔️)|✔️
EightMusesXXX.mjs|https://8muses.xxx|✔️|✔️|(✔️)|✔️
ElevenToon.mjs|http://www.11toon.com|503|✔️|(✔️)|✔️
EmirSUB.mjs|http://emirsub.com|❌|❌|❌|❌
EpikManga.mjs|https://www.epikmanga.com|✔️|✔️|(✔️)|✔️
EvilFlowers.mjs|http://reader.evilflowers.com|✔️|✔️|(✔️)|✔️
ExHentai.mjs|http://exhentai.org|❌|❌|❌|❌
EyeOnManga.mjs|http://www.eyeonmanga.com|❌|❌|❌|❌

### PR DESCRIPTION
**Pull request by XenonR**
Remove connectors identified by #2321 
Closes #2321

### CODE REVIEW

File: src/web/mjs/connectors/AllManga.mjs [hunk 1]
```diff
@@ -1,12 +0,0 @@
-import WordPressMadara from './templates/WordPressMadara.mjs';
-
-export default class AllManga extends WordPressMadara {
-
-    constructor() {
-        super();
-        super.id = 'allmanga';
-        super.label = 'مانجا العرب (M ARAB)';
-        this.tags = [ 'manga', 'arabic' ];
-        this.url = 'https://mangaarab.com';
```
Comment (27136773) [inline src/web/mjs/connectors/AllManga.mjs:10]: Could it be moved to https://www.manhwa.ae, or is this a different website...

File: src/web/mjs/connectors/AtelierDuNoir.mjs [hunk 2]
```diff
@@ -1,49 +0,0 @@
...
-
-    /**
-     *
-     */
-    constructor() {
-        super();
-        super.id = 'atelierdunoir';
-        super.label = 'AtelierDuNoir';
-        this.tags = [ 'manga', 'high-quality', 'english', 'scanlation' ];
-        this.url = 'http://atelierdunoir.org';
```
Comment (27136773) [inline src/web/mjs/connectors/AtelierDuNoir.mjs:16]: New website at https://atelierdunoir.wordpress.com, but only contains a cross link to MangaDex and direct download links... so no need to keep this in HakuNeko

File: src/web/mjs/connectors/ExHentai.mjs [hunk 3]
```diff
@@ -1,13 +0,0 @@
-import EHentai from './EHentai.mjs';
-
-export default class ExHentai extends EHentai {
-
-    constructor() {
-        super();
-        super.id = 'exhentai';
-        super.label = 'ExHentai';
-        this.tags = ['hentai', 'multi-lingual'];
-        this.url = 'http://exhentai.org';
```
Comment (27136773) [inline src/web/mjs/connectors/ExHentai.mjs]: This website is part of the hentai@home network and requires an authorized account to be accessed.
Should not be deleted for now (even so there is currently an IP banning problem for fast downloads...).

**Review by 27136773 — CHANGES_REQUESTED** [review]
E(x)-Hentai should not be deleted

File: src/web/mjs/connectors/AllManga.mjs (hunk 1 above)
Comment (XenonR) [reply src/web/mjs/connectors/AllManga.mjs:10]: I never saw this site, I can only say that https://www.manhwa.ae is a WordpressMadara site. From the website title and URL alone I would say it is a different site. To be safe I propose to add  https://www.manhwa.ae (new connector) while still remove 'allmanga'?

File: src/web/mjs/connectors/ExHentai.mjs (hunk 3 above)
Comment (XenonR) [reply src/web/mjs/connectors/ExHentai.mjs]: I'm unable to access 'exhentai' from anywhere. Are you sure this connector is valid? How do you connect?

File: src/web/mjs/connectors/AllManga.mjs (hunk 1 above)
Comment (27136773) [reply src/web/mjs/connectors/AllManga.mjs:10]: I vote for just removal, user may request Manhwa.ae when needed

File: src/web/mjs/connectors/ExHentai.mjs (hunk 3 above)
Comment (27136773) [reply src/web/mjs/connectors/ExHentai.mjs]: Login with account on their [forum](https://forums.e-hentai.org/) to get a cookie. With this cookie set in your browser the website became available...

image

Comment (XenonR) [reply src/web/mjs/connectors/ExHentai.mjs]: Fair enough.

**Review by 27136773 — APPROVED** [review]
Ready for merge

**Pull request merged**